package.json

```diff
@@ -29,7 +29,7 @@
     "@tailwindcss/forms": "^0.5.6",
     "@tailwindcss/typography": "^0.5.10",
     "@wagmi/chains": "1.8.0",
-    "@wagmi/core": "^1.4.2",
+    "@wagmi/core": "1.4.1",
```

Comment:
@Majorfi we should keep the dependencies pinned, I've ran across patch/minor dependency updates that actually introduce breaking changes.